components/articles/registration-details.tsx

```diff
@@ -0,0 +1,90 @@
+import {Title} from "../infoColumns/infoColumns.styles";
+import {Body, SubTitle} from "../../styles/typography";
+import {MaterialCard} from "../../app/components/elements";
+import {colors} from "../../styles/colors";
+import styled from "styled-components";
+import Link from "next/link";
+
+
+interface PricingTableProps {}
+
+const PricingTable = styled.table<PricingTableProps>`
+  font-size: 1.5rem;
+  margin-left: 3vw;
+  margin-right: 3vw;
+  border: 3px solid ${colors.indigo};
+`;
+
+const Pricing = () => {
+  return <PricingTable>
```

Comment:
should have parenthesis wrapping the component after return ie
return (
    <PricingTable>
         ...........
    </PricingTable>
)